### ISSUE TITLE
Bug in 'cotcurv' mesh_laplacian_smoothing

### ISSUE THREAD

**Issue by shubham-goel** [facebookresearch/pytorch3d#551]
## 🐛 Bugs / Unexpected behaviors
The `cotcurv` method of the `mesh_laplacian_smoothing` function is supposed to compute loss as:
`\sum_j w_ij(u_j - u_i) / (4 A[i])`
where `w_ij = (cot a_ij + cot b_ij)` following reference [1] in the [docs](https://github.com/facebookresearch/pytorch3d/blob/master/pytorch3d/loss/mesh_laplacian_smoothing.py#L9-L83). 

However, the implementation does not follow this, and computes loss as:
`(\sum_j (w_ij u_j) - u_i) / (4 A[i])` https://github.com/facebookresearch/pytorch3d/blob/cd9786e787386c185ef915b3983c5d2861a32907/pytorch3d/loss/mesh_laplacian_smoothing.py#L120
 
These two aren't equivalent as `\sum_j w_ij` doesn't sum-up to 1. 

## Instructions To Reproduce the Issue:
The effect of this bug is evident when you compute the cotcurv laplacian loss for ico-spheres of different levels. Since the 'cotcurv' laplacian is supposed to capture mean curvature, we expect the laplacian loss to be the same at different level ico-spheres. However, that's not the case:
```python
for i in range(5):
    print(mesh_laplacian_smoothing(ico_sphere(i), 'cotcurv'))

# Output:
# tensor(0.1652)
# tensor(1.3907)
# tensor(6.1977)
# tensor(25.4255)
# tensor(102.4149)
```

The implementation can be fixed as:
```python
L_sum = torch.sparse.sum(L, dim=1).to_dense().view(-1, 1)
loss = (L.mm(verts_packed) - L_sum*verts_packed) * norm_w
```
Once fixed, the cotcurv laplacian loss remains same across different ico_sphere levels:
```python
for i in range(5):
    print(mesh_laplacian_smoothing(ico_sphere(i), 'cotcurv'))

# Output:
# tensor(0.3333)
# tensor(0.3354)
# tensor(0.3341)
# tensor(0.3335)
# tensor(0.3334)
```

**Comment by gkioxari** [comment]
I implemented those and there is a high chance you are right. I don't quite remember the details now but I can check them soon. If you want you can submit a PR with the fix and I will review it!

**Comment by shubham-goel** [comment]
Thank you! I've created a PR: #553
